app/models/ability.rb

```diff
@@ -65,6 +65,9 @@ def initialize(user)
 
     # Events
     can [:read, :create], Event
+    can [:participants], Event do |r|
+      user.events.exists?(r.id)
```

Comment:
I think the association ``events`` has a wrong name because it is not clear that the user is organizer!